upup/models/bindata.go

```diff
@@ -3926,15 +3926,15 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
-              - key: "beta.kubernetes.io/os"
+              - key: "kubernetes.io/os"
```

Comment:
I don't think we want to do this just yet as we are only just removing support for k8s 1.9 as of the 1.18 release, and these stable labels were only added as of k8s 1.14.

Looks like we'll need to fix it and all the other uses through the codebase of the beta labels before 1.19 though: https://github.com/kubernetes/kubernetes/pull/91046

Reply:
@gjtempleton This change is applied only to the 1.16+ template, so should be ok.

Reply:
Yep, I’m not planning to backport it to earlier versions since it doesn’t seem to contain any critical fixes, just slight improvements. So if we merge it today it’ll probably go to 1.18 which should be fine.